pkgs/applications/terminal-emulators/xdg-terminal-exec/default.nix

```
@@ -0,0 +1,29 @@
+{ stdenv, lib, fetchFromGitHub, bash }:
+stdenv.mkDerivation rec {
+  pname = "xdg-terminal-exec";
+  version = "0.1.0";
```

```suggestion
  version = "unstable-2023-05-12";
```
https://nixos.org/manual/nixpkgs/stable/#sec-package-naming